tests/dragonfly/__init__.py

```diff
@@ -99,6 +105,16 @@ def create(self, **kwargs) -> DflyInstance:
         self.instances.append(instance)
         return instance
 
+    def start_all(self, instances):
```

Comment:
Maybe we can use [python test containers](https://github.com/boazsade/testcontainers-python) and not run them as sub processes (if I understand correctly, this is what it does here, spins DFs sub processes). They reject my PR for supporting DF, but you can start a DF container with the right parameters.

Reply:
This is inconvenient (increased memory requirements) and requires re-building a container just for testing some change